input/community/speakers/shawn-wildermuth.md

```diff
@@ -18,7 +18,7 @@ Topics:
   - Visual Studio Code
   - Web Development
 Blog: https://wildermuth.com
-Feed: https://wildermuth.com/rss
+Feed: http://feeds.feedburner.com/ShawnWildermuth
```

Comment:
@shawnwildermuth I went ahead and changed the RSS link here to the Feedburner URL so this PR works self-contained. When you go to merge in your 301 support, feel free to change it back.